### PR TITLE
KP-6202 Airflow download demo

### PR DESCRIPTION
**Pull request by helhotti**
Simple Airflow demo for downloading the METS file for a single binding and all ALTO files listed in the downloaded METS.

### CODE REVIEW

File: airflow/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,273 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (aajarven) [inline airflow/docker-compose.yaml:1]: Apache 2.0 license requires us to include attribution notes (NOTICE file) from the original library and to list changes we have made to comply with the license.

**Review by aajarven — CHANGES_REQUESTED** [review]
Works on my machine and the DAG makes sense to me. When we comply with the license for docker-compose.yaml I think we should be good to merge.

File: airflow/docker-compose.yaml (hunk 1 above)
Comment (mmatthiesencsc) [reply airflow/docker-compose.yaml:1]: We have not changed Airflow, though? And we don't plan to distribute Airflow. So I would say stricly the notice file might not be neccessary, but it certainly can't hurt and it is good to be mindful of license issues.
I am not the greatest expert on software licenses, but this TLDR seems to reflect Apache 2.0 quite well: https://tldrlegal.com/license/apache-license-2.0-(apache-2.0)

**Review by mmatthiesencsc — APPROVED** [review]

File: airflow/docker-compose.yaml (hunk 1 above)
Comment (helhotti) [reply airflow/docker-compose.yaml:1]: Let's err on the side of caution, I would say.

**Review by mmatthiesencsc — APPROVED** [review]

**Review by aajarven — APPROVED** [review]
The license compliance changes look good to me and as they also have the Official Martin Seal Of Approval™, I'd say we are good to go.

**Pull request merged**